README.md

```diff
@@ -4,19 +4,17 @@
 [![LICENSE](https://img.shields.io/badge/license-apache2.0-green.svg)](https://github.com/kubernetes-sigs/kubefed/blob/master/LICENSE)
 [![Releases](https://img.shields.io/badge/version-v0.0.10-orange.svg)](https://github.com/kubernetes-sigs/kubefed/releases "Kubefed latest release")
 
-# Kubernetes Cluster Federation
-
-This repo contains an in-progress prototype of some of the
-foundational aspects of V2 of Kubernetes Federation.  The prototype
-builds on the sync controller (a.k.a. push reconciler) from
-[Federation v1](https://github.com/kubernetes/federation/) to iterate
-on the API concepts laid down in the [brainstorming
-doc](https://docs.google.com/document/d/159cQGlfgXo6O4WxXyWzjZiPoIuiHVl933B43xhmqPEE/edit#)
-and further refined in the [architecture
-doc](https://docs.google.com/document/d/1ihWETo-zE8U_QNuzw5ECxOWX0Df_2BVfO3lC4OesKRQ/edit#).
-Access to both documents is available to members of the
-[kubernetes-sig-multicluster google
-group](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster).
+
+Kubernetes Cluster Federation (Kubefed for short) allows you to coordinate the
+configuration of multiple Kubernetes clusters from a single set of APIs in a
```

Comment:
I find `configuration` to be a vague, even possibly confusing term to use. For example `kubeconfig` also holds cluster configuration. What are your thoughts on `coordinate deployment and management of kubernetes applications/resources across multiple kubernetes clusters`?

Reply:
'configuration' in the context of a cluster is intended to indicate the state of the Kubernetes API.  'resources/applications/etc' are specific terms, but fundamentally kubefed is capable of managing any type of configuration in the API.

Reply:
Ack.